src/components/Tabs/Tabs.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react'
+import cn from 'classnames'
+
+import './Tabs.scss'
+
+export const TabsContext = React.createContext()
+
+const Tabs = ({ labels, children, className }) => {
```

Comment:
This component was created for #178 and isn't used anymore.
But it could be useful later, even for the future component library, so I keep it on hand

Reply:
I'll remove the `import` from `index.js` so the bundling ignore it and it won't augment the bundle size

Reply:
That's great. I think a comment at the top of the unused files would useful too.